templates/travis/.travis/before_install.sh.j2

```diff
@@ -91,27 +89,17 @@ fi
 # build (they will be installed as dependencies of the plugin).
 if [ -z "$TRAVIS_TAG" ]; then
 {% endif %}
-  git clone --depth=1 https://github.com/pulp/pulpcore-plugin.git
+git clone --depth=1 https://github.com/PulpQE/pulp-smash.git
 
-  if [ -n "$PULP_PLUGIN_PR_NUMBER" ]; then
-    cd pulpcore-plugin
-    git fetch --depth=1 origin +refs/pull/$PULP_PLUGIN_PR_NUMBER/merge
-    git checkout FETCH_HEAD
-    cd ..
-  fi
-
-
-  git clone --depth=1 https://github.com/PulpQE/pulp-smash.git
-
-  if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
-    cd pulp-smash
-    git fetch --depth=1 origin +refs/pull/$PULP_SMASH_PR_NUMBER/merge
-    git checkout FETCH_HEAD
-    cd ..
-  fi
+if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
```

Comment:
Why was the indentation collapsed?

Reply:
No particular reason other than it bothered me when I saw the output in pulp_file.  I can reverse it